src/index.js

```diff
@@ -1,11 +1,18 @@
 import './style.css';
+
 function component() {
   const element = document.createElement('div');
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = 'Hello Webpack!!!!';
-
+  element.innerHTML = 'Hello Webpack!';
+  element.className = 'hello';
   return element;
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+document.body.appendChild(component());
+
+const footer = document.createElement('div');
+footer.className = 'footer';
+const footerText = document.createElement('p');
+footerText.innerHTML = 'Created By Microverse under CC lisence';
+footer.appendChild(footerText);
+document.body.appendChild(footer);
```

Comment:
I think the footer can be created directly in the HTML file, so the main body will be the one created dynamically.